internal/controller/server_controller_test.go

```diff
@@ -425,49 +421,8 @@ var _ = Describe("Server Controller", func() {
 		// reconciliation as part of handling the available state.
 		Eventually(Object(server)).Should(HaveField("Status.State", metalv1alpha1.ServerStateAvailable))
 
-		zeroCapacity := resource.NewQuantity(0, resource.DecimalSI)
```

Comment:
We should move the processeros status at the top of the test file also to the BMC controller test.